src/index.html

```diff
@@ -5,10 +5,85 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__link nav__link">
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="list__item">
```

Comment:
I think it's an element from the `nav` block also. So you have to name it `nav__list-item` for example.